include/aws/io/io.h

```diff
@@ -14,12 +14,19 @@ AWS_PUSH_SANE_WARNING_LEVEL
 
 #define AWS_C_IO_PACKAGE_ID 1
 
+struct aws_io_handle;
+typedef void aws_io_set_queue_on_handle_fn(struct aws_io_handle *handle, void *queue);
+typedef void aws_io_clear_queue_on_handle_fn(struct aws_io_handle *handle);
+
 struct aws_io_handle {
     union {
         int fd;
+        /* on Apple systems, handle is of type nw_connection_t. On Windows, it's a SOCKET handle. */
```

Comment:
Clarity needed. I don't think we're using nw_connection_t unless dispatch queue is turned on.

"On Apple systems using dispatch queue event loops handle is of type nw_connection_t"